### PR TITLE
add .clang-format file and reformat all .mjs files

### PR DESCRIPTION
**Pull request by fergald**
This is chromium's .clang-format file.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Defines the Chromium style for automatic reformatting.
```
Comment (domenic) [inline .clang-format]: I don't think we should just copy the .clang-format file from Chromium, as these demos are not part of Chromium and won't become so. I think it'd be better to have a minimal file (which seems like it'd just be the first 3 lines?).

Comment (fergald) [reply .clang-format]: done

**Comment by CodeWitchBella** [comment]
Why not use prettier for js formatting?

**Comment by fergald** [comment]
Simply because we're using clang-format elsewhere. If prettier is the preferred formatter for wicg stuff I'm fine with that.

@domenic is it appropriate to drop a `package.json` in here to get prettier?

**Pull request merged**